org.lflang/src/org/lflang/cli/Lfc.java

```diff
@@ -261,27 +261,26 @@ protected Properties filterPassOnProps() {
             BuildParm.SCHEDULER,
             BuildParm.THREADING,
             BuildParm.WORKERS)
-        .map(param -> param.getKey())
+        .map(BuildParm::getKey)
         .collect(Collectors.toUnmodifiableSet());
 
         Properties props = new Properties();
 
         for (OptionSpec option : spec.options()) {
             String optionName = option.longestName();
```

Comment:
```suggestion
            String optionName = option.descriptionKey();
```

Reply:
I think I commented this earlier, but I couldn't see my comment anymore. This issue was fixed on master https://github.com/lf-lang/lingua-franca/pull/1631 in a different way.